packages/guardian/guardian.0.1.0/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Role-based access control for OCaml"
+description:
+  "Defined actors can access targets according to specified rules (allowed actions: CRUD)."
+maintainer: ["engineering@econ.uzh.ch"]
+authors: [
+  "Department of Economics, University of Zurich" "John Christopher McAlpine"
+]
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/uzh/guardian"
+doc: "https://uzh.github.io/guardian"
+bug-reports: "https://github.com/uzh/guardian/issues"
+depends: [
+  "caqti-driver-mariadb" {>= "1.8.0"}
+  "caqti-lwt" {>= "1.8.0"}
+  "containers" {>= "3.6"}
+  "dune" {>= "2.9" & >= "2.9.0"}
+  "logs" {>= "0.7.0"}
+  "lwt" {>= "5.6.1"}
+  "lwt_ppx" {>= "2.1.0"}
+  "mariadb" {>= "1.1.4"}
+  "ocaml" {>= "4.12.0"}
+  "ocamlformat" {>= "0.24.1"}
```

Comment:
I can't find anything in the code that uses ocamlformat. Is this just a dependency that you use personally for development?